docs/conf.py

```diff
@@ -160,7 +161,7 @@
 # html_file_suffix = None
```

Comment:
There is a lot of older documentation stuff that i think we can remove from this file. i plan to do that in my next round of work.